### DISCUSSION TITLE
Information about padlocks

### DISCUSSION THREAD

**Discussion by F8GHE** [meshtastic/Discussions#70] [General]
The function of the padlock remains unclear to me. We are a small group of 3 users with firmware versions higher than 2.5.6 and we randomly have orange open or green closed padlocks.

We also have difficulty with direct messages.

Where can I find explanations on this subject?

**Comment by garthvh** [comment]
![image](https://github.com/user-attachments/assets/6c486e8f-f7c1-48ae-ab42-3ad582c98acc)

**Comment by F8GHE** [comment]
Thanks vert much. Is there a way to switch the padlock from closed to
opened or is it automatic ?

**Reply by garthvh** [reply]
Automatic based on a successful key exchange between the devices

**Reply by fifieldt** [reply]
If it's your first time using firmware above 2.5 you may need to reset your nodeDB to refresh the keys.

**Comment by F8GHE** [comment]
Does this mean that if our padlocks are intermittently open in orange, the
key transfer is not good?
And can you confirm that DMs only go through the primary channel?
73s Christian F8GHE

Le jeu. 14 nov. 2024 à 10:57, Tom Fifield ***@***.***> a
écrit :

**Comment by F8GHE** [comment]
What does an orange closed padlock like this one means ?
![image](https://github.com/user-attachments/assets/69949a71-4626-4059-81e5-ae1a9b1d42ab)

I observe this orange closed padlock when I try to send a DM !
And when I try to send the DM, a cloud crossed is immediatly returned !
Not very easy to understand.
![Screenshot_20241115-090533](https://github.com/user-attachments/assets/680c232c-9e12-4eab-8e06-a903dceff865)

73s Christian F8GHE


Le jeu. 14 nov. 2024 à 13:56, Christian Barthod ***@***.***> a
écrit :

**Reply by F8GHE** [reply]
I have just noticed that with **version 2.5.x**, the node padlock is "**orange open**" when the "**Licensed HamRadio bit**" is set.
In this case the DMs go through **Channel 0**, which seems logical !
73s Christian F8GHE
